patches/llvm/0002-Add-managarm-target.patch

```diff
@@ -557,9 +565,9 @@ index 000000000..3639920fd
 +  };
 +
 +  for (const auto &IncludePath : LibStdCXXIncludePathCandidates) {
-+    if (addLibStdCXXIncludePaths(IncludePath, /*Suffix*/ "", TripleStr,
-+                                 /*GCCMultiarchTriple*/ "",
-+                                 /*TargetMultiarchTriple*/ "",
++    if (addLibStdCXXIncludePaths(IncludePath, /*Suffix*/ TripleStr,
++                                 /*GCCMultiarchTriple*/
++                                 /*TargetMultiarchTriple*/
```

Comment:
The comments now look very strange here.

Reply:
You're right, I'll remove them.